jujugui/static/gui/src/app/init/test-bakery-utils.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2017 Canonical Ltd. */
+'use strict';
+
+const newBakery = require('./init/utils/bakery-utils');
+
+describe('bakery utils', () => {
+  function createNewBakery(
+    config,
```

Comment:
You could define a default value for these as {} instead of doing the || below, just FYI.